.gitignore

```diff
@@ -1,35 +0,0 @@
-# See https://help.github.com/articles/ignoring-files for more about ignoring files.
-#
```

Comment:
Why you removed gitignore file?

Reply:
it got into the commit by mistake, there are no changes relative to the development branch